src/components/element-editor/validation/validation.tsx

```diff
@@ -124,17 +119,67 @@ export const Validation = () => {
     // =========================================================================
 
     const renderChildren = () => {
-        return fields.map((field, index) => (
-            <ValidationChild
-                key={field.id}
-                onDelete={() => handleDelete(index)}
-                options={getValidationOptionsByType(
-                    validationValues,
-                    elementType
-                )}
-                index={index}
-            />
-        ));
+        const options = getValidationOptionsByType(
+            validationValues,
+            elementType
+        );
+
+        return fields.map((field, index) => {
+            switch (elementType) {
+                case EElementType.EMAIL:
+                    return (
+                        <EmailValidationChild
+                            key={field.id}
+                            index={index}
+                            options={options}
+                            disabled={options.length === 1}
+                            onDelete={() => handleDelete(index)}
+                        />
+                    );
+                case EElementType.TEXTAREA:
+                    return (
+                        <LongTextValidationChild
+                            key={field.id}
+                            index={index}
+                            options={options}
+                            disabled={options.length === 1}
+                            onDelete={() => handleDelete(index)}
+                        />
+                    );
+                case EElementType.NUMERIC:
+                    return (
+                        <NumericValidationChild
+                            key={field.id}
+                            index={index}
+                            options={options}
+                            disabled={options.length === 1}
```

Comment:
as admin can add up to 3 rules, this is causing all the rule selection to be disabled from 2nd rule onwards. since admin cant add beyond 3 rules and the added options are removed, we can remove this logic.